### ISSUE TITLE
Scrub command stock in GUI

### ISSUE THREAD

**Issue by waxweazle2001** [Smurf-IV/Elucidate#57]
I changed a HDD in my System in the Drivepool, so the Data on the HDD is moved to the other HHDs in my system.
So I started the scrub command to run the fix after, the program itself seems to run as I see in the Taskmanager but the GUI itself is crashed and don't react to any click. Nothing changed in the Textline and the Windows doesn't react on any click.
Does anyone have the same problem??

**Comment by Smurf-IV** [comment]
What options did you use ?
I'm testing with this:
![image](https://user-images.githubusercontent.com/2418812/71554892-10a6b380-2a1d-11ea-9d47-c57cbf1ec6d3.png)

**Comment by Smurf-IV** [comment]
And now this:
![image](https://user-images.githubusercontent.com/2418812/71554898-46e43300-2a1d-11ea-8fba-18275d95ca50.png)

**Comment by waxweazle2001** [comment]
Hello smurf,
I did a complete new sync so this Problem I can't tell u anything by now. I must to change a HDD to see if the problem persists. I don't have given any Options for the scrub command i only ran the stock mode.
I ran the newest Version you released on a win 10 os.
I don't have any logs as i know. But it could be that the snapraid gui is terminated because of the defect ram memory as i found out 1 week ago. I would say we close this failure because i could not really say if it is the failure of the ram.
Thx for Testing so far.
